localized/games/eggquiz/eggquizmath_levels_en.tsv

```diff
@@ -333,15 +333,15 @@ SpecialTest_14	1	140001~140010, 140001~140010, 140001~140010, 140001~140010, 140
 			140004	word_problem	Hira has 9 pencils. Rashid has 5 more pencils than Hira. How many pencils does Rashid have?	Hira has 9 pencils. Rashid has 5 more pencils than Hira. How many pencils does Rashid have?				14				
 			140005	word_problem	Ganesh has 4 stamps. Daanish has 5 more stamps than Ganesh. How many stamps does Daanish have?	Ganesh has 4 stamps. Daanish has 5 more stamps than Ganesh. How many stamps does Daanish have?				8				
 			140006	word_problem	Jyoti is 5 years old. Her sister Jiya is 2 years older than Jyoti. How old is Jiya?	Jyoti is 5 years old. Her sister Jiya is 2 years older than Jyoti. How old is Jiya?				7				
-			140007	word_problem	There are 14 chairs in Ms. Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?				17				
+			140007	word_problem	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?	There are 14 chairs in Miss Jeet's classroom. There are 3 more chairs in Mr. Nakul's classroom. How many chairs are there in Mr. Nakul's classroom?				17				
```

Comment:
Check for `Mr.`

Reply:
Checked in the TTS, Mr. is spoken correctly by the TTS